### ISSUE TITLE
Error in Group 1 labels

### ISSUE THREAD

**Issue by zalmquist** [statnet/networkDynamic#11]
Per a question from Giacomo, ETCH I notice that the group labels have a small error,

The group labels should be (for vertex.names -- _NOT_ the order of the vertices)

group1<-c(1:8,13, 16:19, 22, 28, 37, 39, 40, 46, 47, 51, 54)
group2<-c(9, 10, 14, 15, 20, 21, 23:27, 29, 32, 33, 41:45, 49, 52)

It looks like 16:19 in Group 1 was dropped.

Thanks!

Zack

**Comment by zalmquist** [comment]
It looks like it is in both formats of the data. Not sure what happend.

**Comment by mbojan** [comment]
Does this apply to the windsurfers data?

**Comment by zalmquist** [comment]
Yes. Both data sets.

Best,

Zack

---
Zack W. Almquist
Assistant Professor, Department of Sociology
Adjunct Assistant Professor, Department of Statistics
Senior Data Scientist Fellow, eScience Institute
Training Core PI, Center for Studies in Demography and Ecology
University of Washington
Editor-in-Chief, Journal of Mathematical Sociology
Website: http://depts.washington.edu/zalmquist

**Comment by skyebend** [comment]
Hi Zach, I'm sorry I'm  not up to speed on context here, looks like 16:19 is on the list for group 1?

**Comment by zalmquist** [comment]
Yeah that is correct. 16:19 should be flagged as part of group 1.

Best,

Zack

---
Zack W. Almquist
Assistant Professor, Department of Sociology
Adjunct Assistant Professor, Department of Statistics
Senior Data Scientist Fellow, eScience Institute
Training Core PI, Center for Studies in Demography and Ecology
University of Washington
Editor-in-Chief, Journal of Mathematical Sociology
Website: http://depts.washington.edu/zalmquist

**Comment by giava90** [comment]
Hi, 
I have noticed that the `vertex.names` match the "order of the vertices":
```
library(networkDynamic)
data("windsurfers")
a<-c(); for (i in 1:95){if(i!=windsurfers$val[[i]]$vertex.names) {a<-c(a,windsurfers$val[[i]]$vertex.names)}}; a
# NULL
```
Also that the nodes 16:18 are flagged as in group1, but 19 and many others are missing:

```
group1<-c(1:8,13, 16:19, 22, 28, 37, 39, 40, 46, 47, 51, 54)
a<-c();for (i in 1:95){if(windsurfers$val[[i]]$group1==1) {a<-c(a,windsurfers$val[[i]]$vertex.names)}};intersect(a, group1)
# [1]  1  2 16 17 18 39 40
```
Could it be possible that indices and `vertex.names` have gotten mixed up somewhere?
I hope this helps to clarify the problem.

Best,
Giacomo

Some code to reproduce the issue:
```
library(networkDynamic)
data("windsurfers")
a<-c();for (i in 1:95){a<-c(a,windsurfers$val[[i]]$group1)}; sum(a)
# [1] 18   The output should be 22
group1<-c(1:8,13, 16:19, 22, 28, 37, 39, 40, 46, 47, 51, 54)
a<-c();for (i in 1:95){if(windsurfers$val[[i]]$group1==1) {a<-c(a,windsurfers$val[[i]]$vertex.names)}}; length(intersect(a, group1))
# [1] 7 The output should be 22
group2 <-c(9, 10, 14, 15, 20, 21, 23:27, 29, 32, 33, 41:45, 49, 52)
a<-c();for (i in 1:95){if(windsurfers$val[[i]]$group2==1) {a<-c(a,windsurfers$val[[i]]$vertex.names)}};length(intersect(a, group2))
# [1] 9  The output should be 21
```
